test/test_pynisher.py

```diff
@@ -260,6 +267,21 @@ def test_crash_unexpectedly(self):
         self.assertEqual(wrapped_function.exit_status, pynisher.SignalException)
         self.assertEqual(wrapped_function.exitcode, 0)
 
+    @unittest.skipIf(not all_tests, "skipping keyboard interruption test")
+    def test_keyboard_interruption(self):
+        print("Testing keyboard interruption.")
+        wrapped_function = pynisher.enforce_limits(
+            context=multiprocessing.get_context(context),
+            logger=self.logger,
+        )(keyboard_interruption)
+        return_value = wrapped_function()
+        self.assertIsNone(return_value)
+        self.assertEqual(wrapped_function.exit_status, pynisher.KeyboardInterruptException)
```

Comment:
Hey, can we somehow test that the subprocess was actually terminated after the `KeyboardInterrupt`?

Reply:
Ok, I will try:)